src/pages/allow-diagnostics/allow-diagnostics-layout.tsx

```diff
@@ -23,43 +24,49 @@ interface AllowDiagnosticsLayoutProps {
   onUserAllowDiagnostics(): void;
   onUserDenyDiagnosticsPermissions(): void;
 }
-export const AllowDiagnosticsLayout: FC<AllowDiagnosticsLayoutProps> = props => {
-  const { onUserAllowDiagnostics, onUserDenyDiagnosticsPermissions } = props;
 
-  const title = 'Help us improve';
+export function AllowDiagnosticsFullLayout(props: AllowDiagnosticsLayoutProps) {
+  const { onUserAllowDiagnostics, onUserDenyDiagnosticsPermissions } = props;
 
   return (
-    <BaseDrawer title={title} isShowing onClose={() => onUserDenyDiagnosticsPermissions()}>
-      <Box mx="loose" mb="extra-loose">
-        <Body>
-          Hiro would like to gather anonymous data to help improve the experience of using Stacks
-          apps and the wallet.
-        </Body>
-        <Stack mt="loose" spacing="base-tight">
-          <ReasonToAllowDiagnostics text="Send anonymous data about page views and clicks" />
-          <ReasonToAllowDiagnostics text="We'll never collect personal data such as your Stacks address, keys, balances, or IP address" />
-          <ReasonToAllowDiagnostics text="We'll never share or sell any user data" />
-        </Stack>
-        <Flex mt="loose" fontSize="14px">
-          <Button
-            type="button"
-            mode="primary"
-            onClick={() => onUserAllowDiagnostics()}
-            mr="base-tight"
-          >
-            Yes, I'll help
-          </Button>
-          <Button
-            type="button"
-            mode="tertiary"
-            ml="base"
-            variant="link"
-            onClick={() => onUserDenyDiagnosticsPermissions()}
-          >
-            No thanks
-          </Button>
-        </Flex>
-      </Box>
-    </BaseDrawer>
+    <PopupContainer header={<Header hideActions />} requestType="auth">
```

Comment:
This is a layout component. We shouldn't add PopupContainer here, as this adds all kinds of global behaviours. Should only be used on the component that has a route pointed to it.

Reply:
I moved the PopupContainer outside of the allow-diagnostics-layout.